introduction.md

```diff
@@ -1,2 +1,8 @@
 Hi! My name is Rich and I love learning Git!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?